pyodide_build/cli/venv.py

```diff
@@ -6,12 +6,114 @@
 from pyodide_build.out_of_tree import venv
 
 
+# TODO: disabled options that can be later supported have been commented out, fix them
+# --copies/--always-copy and symlink_app_data
 def main(
     dest: Path = typer.Argument(
         ...,
         help="directory to create virtualenv at",
     ),
+    clear: bool = typer.Option(
+        False,
+        "--clear/--no-clear",
+        help="Remove the destination directory if it exists",
+    ),
+    no_vcs_ignore: bool = typer.Option(
+        False,
+        "--no-vcs-ignore",
+        help="Don't create VCS ignore directive in the destination directory",
+    ),
+    symlinks: bool = typer.Option(
+        True,
+        "--symlinks",
+        help="Try to use symlinks rather than copies, when symlinks are not the default for the platform",
+    ),
```

Comment:
For which platforms are symlinks not the default? Windows? We don't support windows at all. I suspect if we don't use a symlink it will be broken because it won't copy some necessary files. Let's leave this one out for now.

Reply:
Yes, this is the case for Windows. Though, this option was marked as `True` here and is the default behaviour for Unix-based platforms, so 98c7576086557bf863ca06231d46fe223f94ab2a drops mentions of it in the CLI (so `virtualenv` still uses it, but it isn't overridable now).